docs/development/code_guide.rst

```diff
@@ -5,14 +5,73 @@ Code Development Guidelines
 This document describes the coding requirements and guidelines to be
 followed during the development of PlasmaPy and affiliated packages.
 
-Dependencies
-============
+Code written for PlasmaPy must be compatible with Python 3.6 and
+later. Python 2 is not supported by PlasmaPy.
+
+PlasmaPy requires 
+
+* Astropy 2.0 or later
+* NumPy 1.13 or later
+* scipy 0.19 or later
+* matplotlib 2.0 or later
+
+Setting up an environment for development
+=========================================
+
+To obtain PlasmaPy's source code, use the usual
+
+.. code-block:: bash
+
+  git clone https://github.com/PlasmaPy/PlasmaPy.git 
+
+To be able to import PlasmaPy from your source version:
+
+.. code-block:: bash
+
+  cd PlasmaPy
+  python setup.py develop
```

Comment:
I think it makes more sense to talk about setting up the environment prior to this step. My inclination reading through this is to run the command immediately, and then when I read the next few lines I get a warning about needing to work in a virtual environment. That warning should come earlier.

Reply:
Having thought about that some more - sure, that makes sense. However, we now have the `requirements/base.txt` and `environment.yml`, and those two can be used to facilitate setting up the virtual environment - and the easiest way to get those is going to be via cloning the repo.